--- molokai.vim 2012-08-15 02:26:31.780396257 +0900
+++ molokai.vim 2012-08-15 02:29:12.374546127 +0900
@@ -133,7 +133,7 @@
       hi Normal                   ctermbg=234
       hi CursorLine               ctermbg=235   cterm=none
    else
-      hi Normal       ctermfg=252 ctermbg=233
+      hi Normal       ctermfg=252 ctermbg=none
       hi CursorLine               ctermbg=234   cterm=none
    endif
    hi Boolean         ctermfg=135
